### ISSUE TITLE
max dimension on native

### ISSUE THREAD

**Issue by stevenolen** [ucla/ohmageX#804]
I think the intention was for `maxDimension` on native devices to be set to 800 by default if no `maxDimension` tag was found in the XML, but this code appears to do the opposite (meaning it sets to 800 even if a tag is found):

https://github.com/ucla/ohmageX/blob/develop/www/js/backbone/lib/components/prompts/selector_view_photo.js.coffee#L29-L41

I think the precedence should be (1 most preferred):
1. `maxDimension` in xml
2. `App.custom.prompt_defaults.photo.max_pixels` set for a particular deploy
3. `800` if native and 1/2 are not set. `999999` if non-native and 1/2 are not set.

**Comment by wrenr** [comment]
The intention was to prevent a memory error that happens with large images on native devices, so the constraint could only be 800 max. Testing a version with 1600 for e-QIS


________________________________

**Comment by hongsudt** [comment]
@rrocchio @edsakabu 
Here is the image resolution information that I found from http://www.bhphotovideo.com/FrameWork/charts/resolutionChartPopup.html

Maybe we can set the app maximum resolution to be 2560 (for 5 megapixel)?  Assuming 32 bit per pixel, it will take 2560 x 1920 x 4 which is about 20MB.. This shouldn't crash the app... 
 
1280x960 - 1 Megapixel
1600x1200 - 2 Megapixel
2048x1536 - 3 Megapixel
2240x1680 - 4 Megapixel
2560x1920 - 5 Megapixel
3032x2008 - 6 Megapixel
3072x2304 - 7 Megapixel
3264x2448 - 8 Megapixel

**Comment by stevenolen** [comment]
given the way the app currently handles image data in the upload queue though, 4 images at 5MB per image, then base64 converted and stored in html5 local store will result in attempting to put 26.6MB into 5MB of localstorage right?

**Comment by hongsudt** [comment]
yes.. I am making an assumption that we will have to handle the way we store the image files differently e.g. don't store them in the local storage.

**Comment by stevenolen** [comment]
ok -- there shouldn't really be any issue with sizes in that scenario at all (given we can already handle 1GB video when it is stored/uploaded in this method.
